tkp/db/database.py

```diff
@@ -91,11 +90,13 @@ def __init__(self, **kwargs):
                                              self.host,
                                              self.port,
                                              self.database),
-                                            echo=False
+                                            echo=False,
+                                            poolclass=NullPool,
                                             )
         self.Session = sessionmaker(bind=self.alchemy_engine)
```

Comment:
Sorry, I think you are wrong. `sessionmaker()` is a factory function returning 'configured' `Session` classes. The documentation is not ambiguous about this: http://docs.sqlalchemy.org/en/latest/orm/session_api.html